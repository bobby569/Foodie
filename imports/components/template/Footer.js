import React from 'react';
import { Layout } from 'antd';
const { Footer } = Layout;

export default () => (
	<Footer style={{ textAlign: 'center' }}>Foodie Â© 2018 Created by Foodie Team</Footer>
);
