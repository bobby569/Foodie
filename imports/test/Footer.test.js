import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Footer from '../components/template/Footer';

describe('Footer', () => {
	const footer = shallow(<Footer />);
	const rendered = renderer.create(<Footer />);

	it('Footer should render', () => {
		expect(footer.exists()).toBe(true);
		expect(rendered.toJSON()).toMatchSnapshot();
	});
});
