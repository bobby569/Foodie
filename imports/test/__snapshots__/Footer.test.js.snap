// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer should render 1`] = `
<div
  className="footer ant-layout-footer"
>
  Foodie Â© 2018 Created by Foodie Team
</div>
`;
